Dockerfile

```diff
@@ -1,18 +1,30 @@
-################## BASE IMAGE ######################
-
-FROM clinicalgenomics/mip:latest
+# Base image
+FROM python:3.7
```

Comment:
Consider using the slim variant of this image.

Reply:
Alright! I will give it a try in the next PR. Would like to test the functionality first since there was new code coming in to pypi.